src/main/java/pl/eukon05/eventboard/event/adapter/in/domain/FriendlistAdapter.java

```diff
@@ -0,0 +1,28 @@
+package pl.eukon05.eventboard.event.adapter.in.domain;
```

Comment:
I don't get the package structure. `domain` inside of `adapter`? 🤔

Reply:
I've named the package "domain", because it's for adapters responsible for fetching data from another domains...
Maybe I should make packages for every domain? For example, I could rename this package to "User", because the adapter classes in it connect to the User domain?
The package structure would look like this:
adapter -> in (because it's for loading external data) -> user (because it connects to the User domain)
Do you think that would make sense?

Reply:
I see 🤔 Now it starts to make some sense. If this package structure is useful for you, stay with that. 
I like to use more flat structure, so e.g. `adapter.friends.FriendlistAdapter`, `adapter.x.Y` etc.

Also I don't feel that this is an `in` adapter. We are calling it, and it's going `out` of our domain. I would classify e.g. REST API as `in`, because it calls our domain (from outside)